codespell_lib/data/dictionary.txt

```diff
@@ -4940,6 +4942,8 @@ behaviro->behavior
 behaviuor->behaviour
 behavoir->behavior
 behavoirs->behaviors
+behavoiur->behaviour
+behavoiurs->behaviours
```

Comment:
Perhaps add
```
behavoiural->behavioural
```

Reply:
Done. I added suffixes to the surrounding words.